### ISSUE TITLE
ref is not defined when import.meta.glob is used when template has an html class

### ISSUE THREAD

**Issue by olivervorasai** [nuxt/framework#5650]
### Environment

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v16.11.1`
- Nuxt Version:     `3.0.0-rc.4`
- Package Manager:  `npm@8.10.0`
- Builder:          `vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
------------------------------

### Reproduction

https://stackblitz.com/edit/github-h5grgg?file=app.vue

### Describe the bug

When using `import.meta.globEager` with a `ref`, if the top-level element has a `class`, Nuxt will throw a 500 error stating `ref is not defined`

Here is a simplified version of the code

```vue
<script setup>
const assets = import.meta.globEager('/assets/*.svg', { as: 'raw' });

const someRef = ref('I am a ref');
</script>

<template>
  <div class="this-class-causes-an-error">
  </div>
</template>
```

### Additional context

This may be not the same as https://github.com/nuxt/framework/issues/1693 since I am not using a dynamic path.

### Logs

```
500
ref is not defined

at setup (./.nuxt/dist/server/server.mjs:3400:17)
at _sfc_main.setup (./.nuxt/dist/server/server.mjs:3430:23)
at callWithErrorHandling (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:157:22)
at setupStatefulComponent (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:7084:29)
at setupComponent (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:7039:11)
at renderComponentVNode (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:172:17)
at Module.ssrRenderComponent (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:608:12)
at default (./.nuxt/dist/server/server.mjs:2672:37)
at Module.ssrRenderSuspense (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:946:9)
at _sfc_ssrRender (./.nuxt/dist/server/server.mjs:2667:25)
```

**Comment by danielroe** [comment]
cc: @antfu - might be an unimport issue?

Reproducible with:

```vue
<script setup>
// /*
ref('I am a ref')
</script>

<template>
  <div class="this-class-causes-an-error" />
</template>
```

**Comment by antfu** [comment]
It's already been fixed in unimport v0.3.0, where our rc.4 still using v0.2.

Setting this to use the edge version until rc.5 gets released.

```json
  "devDependencies": {
    "nuxt": "npm:nuxt3@latest"
  }
```
